lib/run_loop/app.rb

```
@@ -67,6 +67,22 @@ def executable_name
       identifier
     end
 
+    # Inspects the app's file for the server version
+    def calabash_server_version
+      unless path.nil? || path == ""
+        path_to_bin = File.join(app_path, executable_name)
+        version_str = xcrun.exec(["strings", path_to_bin])[:out][/CALABASH VERSION: \d+\.\d+\.\d+/, 0]
+        unless version_str.nil? || version_str == ""
+          server_ver = version_str.split(":")[1].delete(' ')
+          RunLoop::Version.new(server_ver)
+        else
+          raise 'App file does not contain Calabash server'
```

I don't think we want to raise an exception.

We should return nil.